src/main/java/Duke.java

```diff
@@ -1,10 +1,193 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LS = System.lineSeparator();
+
+    /* Predetermined Message List */
+    private static final String MESSAGE_LOGO = " ____        _        "+LS
+            + "|  _ \\ _   _| | _____ "+LS
+            + "| | | | | | | |/ / _ \\"+LS
+            + "| |_| | |_| |   <  __/"+LS
+            + "|____/ \\__,_|_|\\_\\___|"+LS;
+    private static final String MESSAGE_INTRO = "\tHello! I'm Duke"
+            + LS +"\tWhat can I do for you?";
+    private static final String MESSAGE_EXIT = "\tBye. Hope to see you again soon!";
+    private static final String MESSAGE_DONE = "\t Nice! I've marked this task as done:";
+    private static final String MESSAGE_TASK_ADDED = "\t Got it. I've added this task:";
+    private static final String MESSAGE_LIST = "\tHere are the tasks in your list:";
+    private static final String MESSAGE_LINE = "\t____________________________________________________________";
+
+
+    /* Command List */
+    private static final String COMMAND_LIST = "list";
+    private static final String COMMAND_DONE = "done";
+    private static final String COMMAND_TODO = "todo";
+    private static final String COMMAND_DEADLINE = "deadline";
+    private static final String COMMAND_EVENT = "event";
+    private static final String COMMAND_EXIT = "bye";
+
+    /* Command Separator Parameter List */
+    private static final String PARAM_DELIMIT_BY = "/by";
+    private static final String PARAM_DELIMIT_AT = "/at";
+    private static final int PARAM_DELIMIT_LIMIT = 2;
+
+    private static Task[] tasks = new Task[100];
```

Comment:
Perhaps you can refractor the number 100 to show the meaning of it since its a constant?